File: Modul-4/Soal-1/PRAK401-2410817110016-MuhammadAzriRahman.py
kelipatan, simbol = input("Masukan Kelipatan dan simbol: ").split()
kelipatan = int(kelipatan)

i = 1
for i in range(1, 51):
    if ((i%kelipatan) == 0):
        print(simbol, end= " ")
    else:
        print(i, end= " ")

File: Modul-4/Soal-1/PRAK401-2410817110016-MuhammadAzriRahman.c
#include <stdio.h>

void main() {
    int kelipatan;
    char simbol;

    printf("Input Kelipatan dan simbol: ");
    scanf(" %d %c", &kelipatan, &simbol);

    for (int i = 1; i <= 50; i++) {
        if ((i % kelipatan) == 0) {
            printf("%c ", simbol);
        } else {
            printf("%d ", i);
        }
    }
}